src/builder-context.c

```diff
@@ -344,6 +346,32 @@ builder_context_find_in_sources_dirs_va (BuilderContext *self,
   return NULL;
 }
 
+void 
+builder_context_set_opt_export_only (BuilderContext *self,
+                                          gboolean       opt_export_only)
+{
+  self->opt_export_only = opt_export_only;
+}
+
+gboolean 
+builder_context_get_opt_export_only (BuilderContext *self)
+{
+  return self->opt_export_only;
+}
+
+void
+builder_context_set_opt_mirror_screenshots_url (BuilderContext *self,
+                                                     char *opt_mirror_screenshots_url)
+{
+  self->opt_mirror_screenshots_url=opt_mirror_screenshots_url;
```

Comment:
This needs to be freed in `builder_context_finalize`, and the function should probably take a `const char*` and `g_strdup` it instead of taking ownership? Also, style nit:

```suggestion
  self->opt_mirror_screenshots_url = opt_mirror_screenshots_url;
```